localPackages/site_developer/Resources/Private/Templates/Page/Default.html

```diff
@@ -0,0 +1,9 @@
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
+    data-namespace-typo3-fluid="true">
+
+    {f:layout(name: 'Default')}
```

Comment:
Why the inline notation here?

Reply:
I always use inline when the result is not a tag, and no area is necessary, which would be necessary when providing a fallback for e.g. an `f:section`, but of course I can change that.